schemas/log.yml

```diff
@@ -43,3 +43,28 @@
       short: Name of the logger.
       description: >
         The name of the logger inside an application. This is usually the name of the class which initialized the logger, or can be a custom name. 
+
+    - name: source.file
```

Comment:
I worry if what we call source here will conflict with what we have in source under https://github.com/elastic/ecs/blob/master/schemas/source.yml At the same time I think some argument could be it is similar?

Reply:
> At the same time I think some argument could be it is similar?

I'd say it's actually not really similar to `source`. We could also rename to `log.origin.*` to avoid ambiguities.

Reply:
+1 on origin.